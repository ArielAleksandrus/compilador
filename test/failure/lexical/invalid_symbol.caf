int func(){
    int a[2+3].;
    retorne 1;
}
programa{}
