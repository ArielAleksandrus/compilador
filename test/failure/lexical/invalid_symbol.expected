ERROR => Line 2, col 15: Unexpected symbol '.'
